pages/chat/_entity.vue

```diff
@@ -70,6 +72,10 @@
     @include ensure-width(300px)
   }
 
+  .simplebar {
+    min-height: 0;
+  }
+
```

Comment:
Why is this here, and not in `reset.scss`, where every other CSS reset is? I'm assuming this is a default simplebar class, and not something you have to add. Either way, it should be in a global CSS stylesheet (and not embedded here)

```suggestion
```